File: plover/meta/command.py
"""Command meta for Stripped Plover.

This handles {PLOVER:command} syntax in translations.
Commands are ignored in stripped plover since there's no engine to command.
"""


def meta_command(ctx, command):
    # In stripped plover, engine commands are no-ops
    action = ctx.copy_last_action()
    action.command = command
    return action

File: src/meta/command.ts
/**
 * Command Meta Handler
 * 
 * Handles {PLOVER:command} syntax in translations.
 */

import { Action, FormatterContext } from '../formatting.js';

/**
 * Handle command meta
 * In stripped plover, engine commands are passed through for handling
 */
export function metaCommand(ctx: FormatterContext, command: string): Action {
  const action = ctx.copyLastAction();
  action.command = command;
  return action;
}
